terraform/cloudfront/response.js

```diff
@@ -13,7 +13,8 @@ function handler(event) {
   headers['x-frame-options'] = { value: 'DENY' };
   headers['x-xss-protection'] = { value: '1; mode=block' };
   headers['x-download-options'] = { value: 'noopen' };
-
+  headers['server'] = { value: '*' };
+  headers['cache-control'] = { value: 'no-cache, no-store, must-revalidate' };
```

Comment:
Mozilla says that `no-store` is most restrictive and other directives are ineffective.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control
```
# conflicted
Cache-Control: private, no-cache , no-store, max-age=0, must-revalidate

# equivalent to
Cache-Control: no-store
```